frontend/src/components/chat/AgentAvatar.css

```
@@ -0,0 +1,103 @@
+
+/* Scale the reused AgentBadge glyph to the avatar box (it defaults to 10px). */
+.agent-avatar__glyph .agent-badge {
+  width: 64%;
+  height: 64%;
+}
+
+.agent-avatar__glyph .agent-badge--fallback {
+  width: auto;
+  height: auto;
+  font-size: 0.72em;
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)